### ISSUE TITLE
Add descriptions of paragraphs 

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_paragraphs#266]
![image](https://user-images.githubusercontent.com/3852805/136393318-6c62493b-923b-430e-8d58-a8e0458bf8f0.png)

It would be good to have useful descriptions of the paragraph types

**Comment by finnlewis** [comment]
Agreed - does anyone know what they are all meant to do? I guess some are quite obvious, but perhaps we should explore / audit  each paragraph and confirm what they are for.

**Comment by msayoung** [comment]
I've put together various things:

Figma file: https://www.figma.com/file/WkubIZ0JtHpPIAiO5DoQ9B/LGD-subsite-components?node-id=0%3A1
Overview of subsites (for the microsite discovery) : https://docs.google.com/document/d/1eRlLPG-NAuwQT7OWzpmSwYZvRkVPiJNkQvvRQfHZZ6c/edit
Docs for devs: https://docs.localgovdrupal.org/devs/features/subsites.html
Docs for content designers: https://docs.localgovdrupal.org/content/features/subsites.html
